policyengine_us/variables/gov/states/hi/tax/income/credits/cdcc/dependent_care_benefit/hi_min_head_spouse_earned.py

```diff
@@ -0,0 +1,52 @@
+from policyengine_us.model_api import *
+
+
+class hi_min_head_spouse_earned(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii minimum income between head and spouse for the CDCC"
+    defined_for = StateCode.HI
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=28"
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=29"
+        "https://files.hawaii.gov/tax/legal/hrs/hrs_235.pdf#page=41"
+        "https://files.hawaii.gov/tax/forms/2022/schx_i.pdf#page=2"
+    )
+
+    def formula(tax_unit, period, parameters):
```

Comment:
please split out the person-level computations into a person-level formula here. if you do that can the formula for this simplify to this?
```python
tax_unit.min(where(head_or_spouse, hi_cdcc_earned_income, np.inf))
```

Reply:
Done! I created `hi_cdcc_earned_income.py` to process all person-level computations.